File: apps/institution-api/Models/InstitutionLocationEmail.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionLocationEmail
    {
        public int InstitutionLocationEmailId { get; set; }
        public int? InstitutionLocationId { get; set; }
        public Guid EmailGuid { get; set; }
        public string EmailUsage { get; set; }
        public string EmailType { get; set; }
        public int? ListSequence { get; set; }
        public string EmailAddress { get; set; }
        public DateTime? EmailBusinessStartDate { get; set; }
        public DateTime? EmailBusinessEndDate { get; set; }

        public virtual InstitutionLocation InstitutionLocation { get; set; }
    }
}

File: apps/planmp/src/libs/common/models/institution-location-email.ts


import { InstitutionLocation } from './institution-location';

export class InstitutionLocationEmail {
    public institutionLocationEmailId: number = 0;
    public institutionLocationId: number;
    public emailGuid: string = '';
    public emailUsage: string = '';
    public emailType: string = '';
    public listSequence: number;
    public emailAddress: string = '';
    public emailBusinessStartDate: Date;
    public emailBusinessEndDate: Date;
    public institutionLocation: InstitutionLocation = new InstitutionLocation();

    constructor(init? : Partial<InstitutionLocationEmail>) {
        
        Object.assign(this, init);
    }
}

